electron/js/lib/openGraph.js

```diff
@@ -28,34 +29,63 @@ const bufferToBase64 = (buffer, mimeType) => {
 };
 
 const fetchImageAsBase64 = url => {
+  const IMAGE_SIZE_LIMIT = 5e6; // 5MB
   return new Promise(resolve => {
-    request({encoding: null, url: encodeURI(url)}, (error, response, body) => {
+    const imageRequest = request({encoding: null, url: encodeURI(url)}, (error, response, body) => {
       if (!error && response.statusCode === 200) {
```

Comment:
can we start avoiding magic numbers?

maybe adding all the status codes in one file and have it like `status.OK`?

Could be part of another PR though..

Reply:
Good point but, yeah, probably not the scope of this PR